pvlib/snowcoverage.py

```diff
@@ -0,0 +1,229 @@
+"""
+The ``snowcoverage`` module contains functions that model snow coverage on
+solar modules. The model was first proposed in Marion et al. 2013 and was
+later validated and implemented into NREL's SAM.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def snow_slide_amount(surface_tilt, sliding_coefficient=1.97):
+    '''
+    Calculates the amount of snow that slides off of the surface of a module
```

Comment:
The way I read the papers, the equation in this function calculates 10 x the fraction of a row's slant height that is uncovered by snow sliding in an hour. A row is an array row, which can be more than one module.  The description as written implies the function returns a mass or volume of snow sliding off a single module.
The 10x aspect can be handled inside this function so that it returns a fraction rather than a number of tenths.
The time aspect needs to be added to the calculation, maybe a parameter 'timestep'